nextcord/ext/commands/bot.py

```diff
@@ -1418,6 +1418,39 @@ async def process_commands(self, message: Message) -> None:
         ctx = await self.get_context(message)
         await self.invoke(ctx)
 
+    async def process_with_str(self, message: Message, content: str) -> None:
+        """|coro|
+
+        This function is like :func:`.process_commands` except it
```

Comment:
![image](https://user-images.githubusercontent.com/34910574/193480663-1e02d36d-ae8d-4d0c-bc8d-643b572962d6.png)
```suggestion
        This function is like :meth:`.process_commands` except it
```